docs/pages/blog/date-pickers-stable-v5.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'The MUI X Date and Time Pickers get a stable v5 release
+description: Migrate to the latest version for improved DX, customizability, and API consistency.
+date: 2022-09-05T00:00:00.000Z
+authors: ['alexfauquette', 'josefreitas']
+tags: ['MUI X', 'News']
+card: true
+---
+
+About four months ago, we moved the date and time pickers from `@mui/lab` and released the first alpha version of the date pickers package. We have focused on improving stability and developer experience since then, and now, we’re happy to announce the first official `@mui/x-date-pickers` [v5 release](https://github.com/mui/mui-x/releases/tag/v5.17.0).
```

Comment:
```suggestion
About four months ago, we moved the date and time pickers from `@mui/lab` and released the first alpha version of the date pickers package.
We have focused on improving stability and developer experience since then, and now, we’re happy to announce the first official `@mui/x-date-pickers` [v5 release](https://github.com/mui/mui-x/releases/tag/v5.17.0).
```